deployments/cluster/educator.nix

```diff
@@ -90,6 +99,8 @@ in
       peer = map (node: address node.config.networking.privateIPv4)
         (attrValues (filterAttrs (name2: node: name != name2 && hasWitnessTag node) nodes));
     };
+
+    after = [ "postgresql.service" ];
```

Comment:
requires?

Reply:
This still needs to be changed to reflect the changed option name.